Library/Homebrew/development_tools.rb

```diff
@@ -34,7 +34,7 @@ def default_cc
     end
 
     def default_compiler
-      :clang
+      :gcc
```

Comment:
I'd rather this was left as `clang` and changed just for Linux.

Reply:
Can do. Since `clang` is the default on macOS`, this change affects only `--generic`.